pp/bare_bones/bit_sequence.h

```diff
@@ -321,6 +325,13 @@ class PROMPP_ATTRIBUTE_PACKED CompactBitSequence : public CompactBitSequenceBase
  public:
   [[nodiscard]] PROMPP_ALWAYS_INLINE BitSequenceReader reader() const noexcept { return {Base::memory_, size_in_bits_}; };
 
+  PROMPP_ALWAYS_INLINE void trim_lower_bytes(uint32_t bytes_count) {
+    assert(Bit::to_bits(bytes_count) <= Base::size_in_bits());
+    memset(Base::memory_, '\0', bytes_count);
+    size_in_bits_ -= Bit::to_bits(bytes_count);
+    memmove(Base::memory_, Base::memory_ + bytes_count, Base::size_in_bytes());
```

Comment:
Heap buffer overflow
```suggestion
    memmove(Base::memory_, Base::memory_ + bytes_count, Base::size_in_bytes() - bytes_count);
```